src/data/alums.json

```diff
@@ -16,5 +16,286 @@
     "github_username": "sahil-shubham",
     "linkedin_username": "",
     "website_url": "https://sahil-shubham.in"
+  },
+  {
+    "name": "Anjay Goel",
+    "github_username": "AnjayGoel",
+    "linkedin_username": "",
```

Comment:
Is there a reason why linkedin usernames were omitted?

Reply:
@cringeyburger can you open an issue for this? We can merge this PR and handle that later.

Reply:
What do you mean by stating that LinkedIn usernames were omitted? 
PS: I opened issue #69 as you requested for default github dp

Reply:
I meant to ask if there is a lore reason why the alums' linkedin usernames were not added but websites were?